test/e2e-go/features/transactions/accountv2_test.go

```diff
@@ -39,6 +40,8 @@ func TestAccountInformationV2(t *testing.T) {
 	proto, ok := config.Consensus[protocol.ConsensusFuture]
 	a.True(ok)
 	os.Setenv("ALGOSMALLLAMBDAMSEC", "200")
+	proto.AgreementFilterTimeoutPeriod0 = 400 * time.Millisecond
+	proto.AgreementFilterTimeout = 400 * time.Millisecond
```

Comment:
if you've set the proto.AgreementFilterTimeoutPeriod0/proto.AgreementFilterTimeout, than you should be able to remove the `os.Setenv("ALGOSMALLLAMBDAMSEC", "200")` and ```defer func() {
 		os.Unsetenv("ALGOSMALLLAMBDAMSEC")
	}()```, right ?